src/liquid/place/move_form.html

```diff
@@ -46,3 +47,29 @@ <h1 class="subtitle">To</h1>
     </div>
   </div>
 </form>
+
+<script type="text/javascript">
+  function handleResponse(event) {
+    console.log('Event', event);
+    if (!event.error) {
+      // Reset form
+      const $form = document.getElementById('form_place_move');
+      $form.reset();
+
+      // Show toast
+      const toastContent = `
+        <p>A worker is taking care of this job, you can please check it 
```

Comment:
```suggestion
        <p>This will be moved shortly!  It may take up to 24 hours to process.
```

Something like this? The language is a bit more focused on the user experience instead of the technology.